lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/grid/query/result/Row.json

```diff
@@ -0,0 +1,21 @@
+{
+	"description": "A single row of a grid query result.",
+	"type": "object",
+	"properties": {
+		"cellValues": {
+			"type": "array",
+			"items": {
+				"type": "object"
```

Comment:
This might not be correct JSON Schema, but if it works, that's not a big deal.

Reply:
"items": {
  "oneOf": [
     {"type": "string"},
     {"type": "number"},
     {"type": "object" }
  ]
},

or omit type altogether - but that might not work with schema-to-pojo